src/core/datatypes/forte_string.cpp

```diff
@@ -68,7 +68,7 @@ int CIEC_STRING::fromString(const char *paValue) {
         i += 2;
         continue;
       } else {
-        char specialSymbol;
+        char specialSymbol = '\0';
         switch (inputString[i + 1]) {
```

Comment:
as you are cleaning up here. Should we move the switch case into an own method? this would make the code much more readable.